PA_Feature_Detecting.md

```diff
@@ -0,0 +1,70 @@
+# Feature detecting Protected Audience
+
+As Chrome ships and experiments with various Protected Audience APIs, it can be useful for web developers to detect the presence of individual features
+in users’ browsers.  This document seeks to list feature detection mechanisms for each shipped or experimented feature.
+
+## Protected Audience
+[Intent to Ship](https://groups.google.com/a/chromium.org/g/blink-dev/c/igFixT5n7Bs/m/ZNrDcQ2dDQAJ)
+```
+let pa = typeof navigator.joinAdInterestGroup !== 'undefined';
```

Comment:
Will this fail if a publisher site that runs PAA auctions has disabled `joinAdInterestGroup` via Permission-Policy?
In that case should the publisher-side code check

`let pa = typeof navigator.runAdAuction !== 'undefined';`